TUNIC: Add note about bushes to logic section of game info page (#3555)
ArchipelagoMW/Archipelago: worlds/tunic/docs/en_TUNIC.md

## What should I know regarding logic?
In general:
- Nighttime is not considered in logic. Every check in the game is obtainable during the day.

> - Bushes are not considered in logic. It is assumed that the player will find a way past them, whether it is with a sword, a bomb, fire, luring an enemy, etc. There is also an option within the randomizer settings -> general settings to clear some early bushes.

Reviewer: I would specify that this setting only exists in-game, something like this for example
```suggestion
- Bushes are not considered in logic. It is assumed that the player will find a way past them, whether it is with a sword, a bomb, fire, luring an enemy, etc. There is also an option in the in-game randomizer settings menu to clear some of the early bushes.
```
